main.cpp

```diff
@@ -52,84 +52,103 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       driver
+//  action 1:   the driver slows down
+driver.slowDown();
+//  action 2:   the driver drives
+driver.drive();
+//  action 3:   the driver stops
+driver.stop();
+
+
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       robot
+//  action 1:   the robot moves
+robot.move();
+//  action 2:   the robot talks
+robot.talk();
+//  action 3:   the robot rotates
+robot.rotate();
+
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       dog
+//  action 1:   the dog barks
+dog.bark();
+//  action 2:   the dog eats
+dog.eat();
+//  action 3:   the dog sleeps
+dog.sleep();
+
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       movie
+//  action 1:   the movie begins
+movie.begin();
+//  action 2:   the movie ends
+movie.end();
+//  action 3:   the movie pauses
+movie.pause();
+
+
+//  5)          
+//  Noun:       boat
+//  action 1:   the boat spins
+boat.spin();
+//  action 2:   the boat sails
+boat.sail();
+//  action 3:   the boat sinks
+boat.sink();
+
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       hand
+//  action 1:   the hand touches
+hand.touch();
+//  action 2:   the hand shakes
+hand.shake();
+//  action 3:   the hand rotates
+hand.rotate();
+
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       phone
+//  action 1:   the phone rings
+phone.ring();
+//  action 2:   the phone dials
+phone.dial();
+//  action 3:   the phoe falls
+phone.fall();
+
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       bird
+//  action 1:   the bird flies
+bird.fly();
+//  action 2:   the bird sings
+bird.sing();
+//  action 3:   the bird chirps
+bird.chirp();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.